gammapy/modeling/parameter.py

```diff
@@ -483,6 +514,17 @@ def check_limits(self):
         for par in self:
             par.check_limits()
 
+    @property
+    def prior(self):
+        return [par.prior for par in self]
+
+    def prior_stat_sum(self):
+        parameters_stat_sum = 0
+        for par in self:
+            if par.prior is not None:
+                parameters_stat_sum += par.prior(par)
```

Comment:
This looks a bit weird to write `par.prior(par)`. The prior set on a parameter should always evaluate on the parameter value, no?